charts/super-agent-deployment/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: super-agent-deployment
+description: A Helm chart to install New Relic Super agent on Kubernetes
+
+type: application
+version: 0.0.0-beta
+appVersion: nightly  # Change this with a proper version of the image.
```

Comment:
can you add TODO? just to be consistent?

Reply:
🤦

Reply:
Done in [#2e7ea0a](https://github.com/newrelic/helm-charts/pull/1155/commits/2e7ea0a72b79ef91f8fdc94f8231798b34553883)